Sources/GRPCCore/Documentation.docc/Tutorials/Hello-World/Hello-World.tutorial

```diff
@@ -28,17 +28,28 @@
   @Section(title: "Run a gRPC application") {
     Let's start by running the existing Greeter application.
 
+    As a prerequisite you must have the Protocol Buffers compiler (`protoc`) installed. You can
+    find the instructions for doing this in the [gRPC Swift Protobuf
+    documentation](https://swiftpackageindex.com/grpc/grpc-swift-protobuf/documentation/grpcprotobuf/installing-protoc).
+    The remainder of this tutorial assumes you installed `protoc` and it's available in
+    your `$PATH`.
+
+    You may notices that the `swift` commands are all prefixed with `PROTOC_PATH=$(which protoc)`,
+    this is to let the build system know where `protoc` is located so that it can generate stubs
+    for you. You can read more about it in the  [gRPC Swift Protobuf
+    documentation](https://swiftpackageindex.com/grpc/grpc-swift-protobuf/documentation/grpcprotobuf/generating-stubs).
```

Comment:
Note that this is a pre-emptive change: I think we should move the generating stubs doc to the protobuf package and then link to it from the core package docs.